android/core/src/main/java/com/mongodb/stitch/android/core/internal/StitchAppClientImpl.java

```diff
@@ -305,11 +307,39 @@ public void onAuthEvent(final StitchAuth auth) {
       if (binder == null) {
         this.serviceClients.remove(weakReference);
       } else {
-        binder.onRebindEvent();
+        binder.onRebindEvent(rebindEvent);
       }
     }
   }
 
+  @Override
+  public void onAuthEvent(StitchAuth auth) {
+  }
+
+  @Override
+  public void onUserLoggedIn(final StitchAuth auth,
+                             final StitchUser loggedInUser) {
+    onRebindEvent(new AuthEvent.UserLoggedIn<>(loggedInUser));
```

Comment:
Should you be rebinding on all of these?

Reply:
Yes, it is up to the service to determine what to do with the information given.